### DISCUSSION TITLE
npx cap add ios failing at update ios

### DISCUSSION THREAD

**Discussion by INC01067** [ionic-team/capacitor#6046] [Q&A]
Hi experts,

I am trying to convert a ReactJS app to PWA iOS compatible using Capacitor JS as per the steps in below links:
1. [https://medium.com/how-to-react/convert-your-existing-react-js-app-to-android-or-ios-app-using-the-ionic-capacitor-a127deda75bd](https://medium.com/how-to-react/convert-your-existing-react-js-app-to-android-or-ios-app-using-the-ionic-capacitor-a127deda75bd)
2. [https://capacitorjs.com/docs/ios](https://capacitorjs.com/docs/ios)

However, I am getting below error and not able to figure out the reason

![image](https://user-images.githubusercontent.com/54398335/199500372-b270ee2c-bfff-4718-a0b9-28ac44b591e1.png)

**Comment by sdetweil** [comment]
had to create new app and migrate source and configs...
